litmus-portal/frontend/cypress/integration/examples/actions.spec.js

```diff
@@ -0,0 +1,299 @@
+/// <reference types="cypress" />
```

Comment:
@Saranya-jena Can you remove this Examples folder itself,I don't think we are using it.